sortinghat/cmd/unify.py

```diff
@@ -58,6 +62,8 @@ def __init__(self, **kwargs):
                                  help="do not rigorous check of values (i.e, well formed email addresses)")
         self.parser.add_argument('-i', '--interactive', action='store_true',
                                  help="run interactive mode while unifying")
+        self.parser.add_argument('-l', '--log-path', dest='log_path', default=None,
```

Comment:
In my opinion, the user shouldn't be able to define the matches file. I think is something internal to this command. What the user can do is whether it want to use the resume/recovery mode or not.
The file should be always stored in the same place.